src/modules/blockchainApplication/manage/store/selectors.js

```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line import/prefer-default-export
```

Comment:
I think this eslint ignore can be removed now since there are two named exports here

Reply:
the rule say its need to have export default it doesn't care how many exports it has

Reply:
I think if there are more than one named exports there would be no need for this rule `import/prefer-default-export` excpetion.